texastribune/donations
# Small improvements and formatting fixes

#### What's this PR do?
This is part of ongoing efforts to get our front-end code in tip-top shape for Ashley's arrival.

+ Addresses a few of the "we should do this later" bullet points left in my comment on [this PR](https://github.com/texastribune/donations/pull/60) (including making `SelectList` consistent with the other store-driven form elements)
+ Fixes various formatting and linting inconsistencies
+ Renames and moves a few files

I'm not changing any logic.

#### How should this be manually tested?
+ `make` then `make restart`
+ Ensure you can successfully give on `/business`, `/donate`, `/circle` and `/blastform`

#### How should this change be communicated to end users?
NA.

#### Are there any smells or added technical debt to note?
Less!

#### What are the relevant tickets?
Off-sprint.

#### Have you done the following, if applicable:
